base_tier_validation/wizard/comment_wizard.py

```diff
@@ -0,0 +1,43 @@
+# Copyright 2019 Ecosoft Co., Ltd. (http://ecosoft.co.th)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class CommentWizard(models.TransientModel):
+    _name = 'comment.wizard'
+    _description = 'Comment Wizard'
+
+    validate_reject = fields.Char()
+    res_model = fields.Char()
+    view_id = fields.Many2one(
```

Comment:
After I try to understand it, it looks like just a res_id as integer. Nothing related to ir.ui.view, is it???
```suggestion
    res_id = fields.Integer(
```

Reply:
Edited